public/js/admin.js

```diff
@@ -128,6 +128,20 @@ $(document).ready(function(){
         var url       = "/dashboard/user/create";
         var token     = $("#_token").val();
         var username  = $("#username").val();
+
+        //throw error if nothing was passes in the username field
+        username = username.trim();
```

Comment:
**Code Quality - Nitpick**

Avoid duplication. The trim method could have been applied on line 130 like so:
```js
var username = $("#username").val().trim();
```